src/karts/kart.cpp

```diff
@@ -3092,12 +3092,20 @@ const Vec3& Kart::getNormal() const
 }   // getNormal
 
 // ------------------------------------------------------------------------
-/** Returns the position 0,25s before */
+/** Returns the position 0.25s before */
 const Vec3& Kart::getPreviousXYZ() const
 {
     return m_previous_xyz[29];
```

Comment:
As hiker previously mentionned, a constant would be preferable (here and everywhere related to m_previous_xyz).